safe_transaction_service/history/services/index_service.py

```diff
@@ -280,10 +281,13 @@ def reindex_master_copies(
         while reindexing
 
         :param from_block_number: Block number to start indexing from
+        :param to_block_number: Block number to stop indexing on
         :param block_process_limit: Number of blocks to process each time
         :param addresses: Master Copy or Safes(for L2 event processing) addresses. If not provided,
             all master copies will be used
         """
+        assert (not to_block_number) or to_block_number > from_block_number
```

Comment:
Given that this statement won't be available if we run with the optimiser flag should we change this to `raise` an exception instead?